java/src/jmri/jmrit/XmlFile.java

```diff
@@ -571,14 +624,58 @@ static public String xmlDir() {
         return FileUtil.getProgramPath() + "xml" + File.separator;
     }
 
-    static public boolean getVerify() {
-        return verify;
+    
+    /** Whether to, by global default, validate the file being read.
+     * Public so it can be set by scripting and for debug
+     */ 
+    static public Validate getDefaultValidate() {
+        return defaultValidate;
+    }
+
+    static public void setDefaultValidate(Validate v) {
+        defaultValidate = v;
+    }
+ 
+    static private Validate defaultValidate = Validate.None;
```

Comment:
That's the same default as now.  The ConfigurationManager (for panel files, etc) defaults to validating internally, overriding this.